scripts/release/download_ERA5_TEC.py

```diff
@@ -0,0 +1,331 @@
+                if tropo_model in ["ERA5", "ERAINT"]:
+                    pa.ECMWFdload(
+                        date_list2dload,
+                        hour,
+                        grib_dir,
+                        model=tropo_model,
+                        snwe=snwe,
+                        flist=grib_files2dload,
+                    )
+
+                elif tropo_model == "MERRA":
+                    pa.MERRAdload(date_list2dload, hour, grib_dir)
+
+                elif tropo_model == "NARR":
+                    pa.NARRdload(date_list2dload, hour, grib_dir)
+            except Exception:
+                if i < 3:
+                    logger.warning(f"The {i} attempt to download failed, retrying it.")
+                else:
+                    logger.error("Downloading failed for 3 times. Stopping.")
+
+    # check potentially corrupted files
+    grib_files = check_exist_grib_file(grib_files)
+    return grib_files
+
+
+def check_pyaps_account_config(tropo_model: str, parent_dir: Path):
+    """Check for input in PyAPS config file.
+
+    Parameters
+    ----------
+    tropo_model :str
+        tropo model being used to calculate tropospheric delay
+    parent_dir : Path
+        Path to the PyAPS package where it is installed
+
+    Raises
+    ------
+    ValueError
+        If account info is not set in PyAPS config file.
+    """
+    # Convert MintPy tropo model name to data archive center name
+    # NARR model included for completeness but no key required
+    MODEL2ARCHIVE_NAME = {
+        "ERA5": "CDS",
+        "ERAI": "ECMWF",
+        "MERRA": "MERRA",
+        "NARR": "NARR",
+    }
+    SECTION_OPTS = {
+        "CDS": ["key"],
+        "ECMWF": ["email", "key"],
+        "MERRA": ["user", "password"],
+    }
+
+    # Default values in cfg file
+    default_values = [
+        "the-email-address-used-as-login@ecmwf-website.org",
+        "the-user-name-used-as-login@earthdata.nasa.gov",
+        "the-password-used-as-login@earthdata.nasa.gov",
+        "the-email-adress-used-as-login@ucar-website.org",
+        "your-uid:your-api-key",
+    ]
+
+    # account file for pyaps3 < and >= 0.3.0
+    cfg_file = parent_dir / "model.cfg"
+    rc_file = Path("~").home() / ".cdsapirc"
+
+    # for ERA5: ~/.cdsapirc
+    if tropo_model == "ERA5" and rc_file.is_file():
+        pass
+
+    # check account info for the following models
+    elif tropo_model in ["ERA5", "ERAI", "MERRA"]:
+        section = MODEL2ARCHIVE_NAME[tropo_model]
+
+        # Read model.cfg file
+        cfg_file = parent_dir / "model.cfg"
+        cfg = ConfigParser()
+        cfg.read(cfg_file)
+
+        # check all required option values
+        for opt in SECTION_OPTS[section]:
+            val = cfg.get(section, opt)
+            if not val or val in default_values:
+                msg = "PYAPS: No account info found "
+                msg += f"for {tropo_model} in {section} section in file: {cfg_file}"
+                raise ValueError(msg)
```

Comment:
is this something from mintpy? just trying to figure out what the `model.cfg` file is but not finding it on associated with ERA5

Reply:
This used to be where the credentials are stored for PyAPS, some people may still use it